addons/website_event/static/src/scss/website_event.scss

```diff
@@ -1,13 +1,219 @@
-.o_wevent_participating_container {
-    position: relative;
+#o_we_search_box {
+    flex: 0 0 100%;
+    .form-control {
+        @include hover-focus {
+            border-color: $primary;
+            box-shadow: none;
+        }
+    }
+    @include media-breakpoint-up(lg) {
+       flex: 0 0 33%;
+    }
+}
+
+// TODO: Improve this part
+#o_we_index_topbar_filters {
+    .dropdown-toggle {
+        display: block;
+        border: 1px solid #ced4da;
```

Comment:
Sure there is not a bootstrap variable that should be used here ?

Reply:
Yes, i don't like this neither. I will try to find a better solution. This is one of my TODO.